cm_generate_makefile.bash

```diff
@@ -580,7 +580,6 @@ if [ "${KOKKOS_PATH}" == "" ]; then
   KOKKOS_PATH=`dirname $CM_SCRIPT`
   KOKKOS_PATH="${KOKKOS_PATH}/../kokkos"
   echo "CHECKING: $KOKKOS_PATH"
```

Comment:
@brian-kelley I think this is cruft from when I was originally working on the script, can you delete this line (582) as well? Thanks for the clean up, no need to retest.